src/maxKey.js

```diff
@@ -8,13 +8,25 @@
  * @param {object} array - A non-empty array of numbers
  * @returns {number} The key with the largest value
  */
+
+
 function maxKey(obj) {
-  // This is your job. :)
+  var dict = {};
+var arr = [];
+for (var key in dict) {
+    if (dict.hasOwnProperty(key)) {
+        arr.push( [ key, dict[key] ] );
```

Comment:
If for some reason you want an array of `[key, value]` pairs, you can call `Object.entries(obj)`.

That said, you probably don't want that here. Use `Object.keys` to get all the keys. Iterate over the keys one at a time. Keep track of the key with the largest value so far. Compare each key/value pair you see to the largest key/value pair you've seen so far. If larger, replace largest so far.